### DISCUSSION TITLE
new view, seperate monitors

### DISCUSSION THREAD

**Discussion by kian25** [jupyterlab/jupyterlab#14291] [Ideas]
The new view feature is really nice on jupyterlab, but on the desktop or the browser version it's not possible to drag it to a different monitor which in some ways defeats the purpose. 

Would love the ability to do this, having the same file opened twice isn't the same thing as the constant syncing causes issues

**Comment by krassowski** [comment]
As a workaround you can use [RTC](https://jupyterlab.readthedocs.io/en/stable/user/rtc.html) mode to have content synced between two windows.
